celesta-test/src/test/java/ru/curs/celesta/dbutils/adaptors/OraAdaptorTest.java

```diff
@@ -20,7 +20,7 @@
         Locale.setDefault(Locale.US);
     }
 
-    public static OracleContainer<?> oracle = new OracleContainer<>();
+    public static OracleContainer oracle = new OracleContainer();
```

Comment:
Again, why was <?> generic removed?

Reply:
Again, because OracleContainer is not a parameterized class anymore.